test/common/vmmanage.py

```diff
@@ -97,9 +97,12 @@ def build_and_install(install_image, build_image, args):
     args.setdefault("containers", False)
     args.setdefault("address", None)
     try:
-        skip = "cockpit-ostree"
-        if install_image and "fedora-atomic" in install_image:
-            skip = None
+        skips = [ ]
+        if install_image and "fedora-atomic" not in install_image:
```

Comment:
Why just fedora-atomic? This seems to be a relic - I suggest just `atomic` to also cover RHEL Atomic Host.